src/config/config-validator.ts

```diff
@@ -291,7 +291,7 @@ export const validate = function (config: Object): void {
   const valid = validator(config)
 
   if (!valid) {
-    const output = (betterAjvErrors(schema, config, validator.errors, { format: 'js' }) as never as betterAjvErrors.IOutputError[])
+    const output = betterAjvErrors(schema, config, validator.errors ?? [], { format: 'js' })
```

Comment:
The new version returns the expected type, thanks to `{ format: 'js' }` in the options, so no cast is needed.